tools/torch/toGraph.lua

```diff
@@ -0,0 +1,260 @@
+-- Copyright (c) 2015-2016, NVIDIA CORPORATION. All rights reserved.
+require 'torch'
+require 'pl'
+
+opt = lapp[[
+-n,--network (default '') path to network file
+-o,--output (default '') output path of model_def.json
+]]
```

Comment:
Neat! How hard would it be for you to include parameters the output json per node? Why is the type information included in both {'source' and 'target'} of the link and the {node} itself?
Also at this point maybe check that the `network` and `output` options are defined at all (for `cli` usage).